scania/corporate-ui-site
# Feature - New layout

**Describe pull-request**  
_Describe what the pull-request is about_
- Adds new grid package
- Setting up the layout
- Removing old grid system parts
- Cleaning up components in the site
- Added header, content-header and sidenav


**How to test**  
_Add description how to test if possible_
- Run this site
- Try it out in different breakpoints
- Sidebar/grid should work as intended


This is the intial part for the layout(components and the data in them might now work as intended)
